CHANGES.txt

```diff
@@ -1,3 +1,253 @@
+stratisd 3.7.0
+==============
+Recommended Rust toolchain version: 1.81.0
+Recommended development platform for Python development: Fedora 39
+
+* indicates new since stratisd 3.6.8
+
+* Increase D-Bus revision number to 7:
+  https://github.com/stratis-storage/stratisd/pull/3483
+
+* Add support for snapshot revert operation:
+  https://github.com/stratis-storage/stratisd/issues/3621
+  https://github.com/stratis-storage/stratisd/issues/3622
+  https://github.com/stratis-storage/stratisd/pull/3694
+
+* Add origin field to StratFilesystem:
```

Comment:
Is this correct?  StratFilesystem?

Reply:
Better would be:

* Store origin in filesystem metadata and expose it as a D-Bus property: